src/taskActionsHelper.js

```diff
@@ -0,0 +1,52 @@
+export const addTask = (description, taskList) => {
+  const taskIndex = taskList.length;
+
+  const task = {
+    description,
+    completed: false,
+    index: taskIndex,
+  };
+
+  taskList.push(task);
+
+  return taskList;
+};
+
+export const editTask = (description, taskIndex, taskList) => {
+  let theTask = null;
+  let taskPosition = null;
+  taskList.map((task, pos) => {
+    if (task.index === parseInt(taskIndex, 10)) {
+      theTask = task;
+      taskPosition = pos;
+    }
+    return 0;
+  });
+
+  theTask.description = description;
+  taskList[taskPosition] = theTask;
+
+  return taskList;
+};
+
+export const deleteTask = (taskIndex, taskList) => {
+  taskList = taskList.filter((task) => task.index !== parseInt(taskIndex, 10));
+
+  taskList.map((task, pos) => {
+    task.index = pos;
+    return 0;
```

Comment:
* Kindly add the delete functionality to your UI( user interface) as currently, i can't delete a task [Check project requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#:~:text=Implement%20a%20function%20for%20deleting%20a%20task%20(remove%20an%20element%20from%20the%20array).)

Reply:
Hello Reviewer, thanky for your review, the deleteTask functionnality is added, you need to click on a task description in order to see the 'bin' icon that will delete the task when you click on it. I followed the requirements, kindly take time to check again. Thank you.